### DISCUSSION TITLE
How the crestereo_combined_xxx models were created?

### DISCUSSION THREAD

**Discussion by daviszhu88** [PINTO0309/PINTO_model_zoo#356] [General]
The models downloaded cause crash in ONNX C++ runtime. I regenerated some models like "crestereo_init_xxx" and "crestereo_next_xxx" and were loaded successfully. I am wondering how the "crestereo_combined_xxx" models were created? Thanks.

**Comment by PINTO0309** [comment]
https://github.com/PINTO0309/PINTO_model_zoo/blob/main/284_CREStereo/onnx_merge.py

**Reply by daviszhu88** [reply]
Thanks. That worked.

**Reply by daviszhu88** [reply]
Unfortunately, the regenerated model still leads to crash.

Anyone see this message had successfully load the model in C++ with ONNX runtime and get correct results? Thanks.
